universal-application-tool-0.0.1/app/modules/SecurityModule.java

```diff
@@ -122,10 +113,10 @@ protected Config provideConfig(GuestClient guestClient, FormClient formClient) {
                 forbidden("403 forbidden").as(HttpConstants.HTML_CONTENT_TYPE)));
     Config config = new Config();
     config.setClients(clients);
-    config.addAuthorizer("uatadmin", new RequireAllRolesAuthorizer("ROLE_UAT_ADMIN"));
-    config.addAuthorizer("programadmin", new RequireAllRolesAuthorizer("ROLE_PROGRAM_ADMIN"));
-    config.addAuthorizer("applicant", new RequireAllRolesAuthorizer("ROLE_APPLICANT"));
-    config.addAuthorizer("intermediary", new RequireAllRolesAuthorizer("ROLE_TI"));
+    config.addAuthorizer("uatadmin", new RequireAllRolesAuthorizer(Roles.ROLE_UAT_ADMIN));
+    config.addAuthorizer("programadmin", new RequireAllRolesAuthorizer(Roles.ROLE_PROGRAM_ADMIN));
```

Comment:
question (non-blocking): Are we including the program admin role just to make the UAT extensible to allow for program admin-specifics in the future? Just clarifying that our MVP doesn't include the program admin role as being distinct from the UAT admin role.

Reply:
Oh, I'll remove that then.

Reply:
(ah, now I can comment on this thread again). We chatted offline and realized we actually do need this role - my bad! Resolving this thread.